_layouts/home.html

```diff
@@ -0,0 +1,47 @@
+<!doctype html>
+<html lang="en">
+<head>
+{% include common-head.html %}
+</head>
+
+<body>
+{% include common-nav.html %}
+
+{% include home-slides.html %}
+{% include home-empowering.html %}
+{% include home-subscribe.html %}
+{% include home-video.html %}
+
+{% include common-footer.html %}
+{% include common-scripts.html %}
+
+<script src="{{baseurl}}/assets/vendor/owlcarousel2-2.3.4-dist/owl.carousel.js"></script>
+
+<script type="text/javascript">
+    var _gaq = _gaq || [];
```

Comment:
Same here (don't mix HTML and Js and Css in new code)

Reply:
This is a layout, it must have the html template (html + css + js)
It only is used for home page, so I haven't create includes for its specific code